lib/octokit/client/pull_requests.rb

```diff
@@ -98,7 +98,7 @@ def create_pull_request_for_issue(repo, base, head, issue, options = {})
       # @example
       #   @client.update_pull_request('octokit/octokit.rb', 67, title: 'new title', body: 'updated body', state: 'closed')
       # @example Empty body by passing empty string
-      #   @client.update_pull_request('octokit/octokit.rb', 67, body: '')
+      #   @client.update_pull_request('octokit/octokit.rb', title: '67', body: '')
```

Comment:
Looks like there are other occurrences that could be updated as well. Thanks https://github.com/octokit/octokit.rb/blob/main/lib/octokit/client/pull_requests.rb#L99-L103